mm2src/mm2_main/src/lp_swap/pubkey_banning.rs

```diff
@@ -47,23 +56,33 @@ pub async fn list_banned_pubkeys_rpc(ctx: MmArc) -> Result<Response<Vec<u8>>, St
 struct BanPubkeysReq {
     pubkey: H256Json,
     reason: String,
+    duration_min: Option<u8>,
```

Comment:
a maximum of `4 hours and 15 mins` is unnecessarily restricting.

Reply:
I don't know why I thought that was enough tbh. I will set it to highest possible uint.